edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -46,7 +46,8 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             'course_run': True,
             'marketing_url': url,
             'course_id': course['key'],
-            'course_run_id': course_run['key']
+            'course_run_id': course_run['key'],
+            'marketable': 1,  # Mark the course_run to be marketable when updated through this script
```

Comment:
Can we align all of the Boolean values to the same standard—preferably, `True` or `False`? The course_run field above is using a Boolean value. It sounds like marketable and activated are using integers.